assets/js/blocks-registry/block-components/register-block-component.js

```diff
@@ -24,20 +24,24 @@ const assertOption = ( options, optionName, expectedType ) => {
  *
  * @export
  * @param {Object}   options           Options to use when registering the block.
- * @param {string}   options.main      Name of the parent block.
+ * @param {string}   [options.parent]  Name of the parent block, or blank to register globally.
```

Comment:
Are we using `parent` anywhere for real? (I see it in the tests.) Is this needed for something in a follow up, i.e. are we planning to use it?

Reply:
It could be used if you wanted a different component to handle a specific parent block, plus it was from the old API so kept it around.